lua/git-dashboard-nvim/git.lua

```diff
@@ -35,17 +47,9 @@ Git.get_commit_dates = function(username, _branch)
 
   local since_date = os.date("%Y-%m-%d", os.time({ year = current_year, month = 1, day = 1 }))
 
-  local branch = _branch
-
-  if _branch == "main" then
-    branch = _branch
-  else
-    branch = branch .. " --not origin/main"
```

Comment:
I think we could allow the user to pass custom flags from their config, and we could add an example of passing this flag. 
The purpose of it is so that when you create a branch from main, in the new branch you don't see the commits from main, but you "start from scratch", so that the chart is more representative.

Reply:
I see, makes sense!
I'll revert this and search for an existing master/main branch